README.md

```diff
@@ -1,70 +1,192 @@
+# 🧑‍💻 Fruit <a name="about-project"></a>
 
-### `npm test`
+> ** The Fruit Catalog Application is a web-based application that allows users to browse and explore various fruits along with their nutritional information. Users can view a list of fruits, filter them based on their names, and click on individual fruits to view detailed information about each fruit. The application integrates with an external API to fetch fruit data, including images and nutritional content.**
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm run build`
+### Tech Stack <a name="tech-stack"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+    <li>Redux Toolkit</li>
+  </ul>
+</details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Not Available</li>
+  </ul>
+</details>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Not Available</li>
+  </ul>
+</details>
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+<!-- Features -->
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Key Features <a name="key-features"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- List of Fruits Page
+- Details page
+- Nutritions information
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Learn More
+<!-- LIVE DEMO -->
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## 🚀 Live Demo <a name="live-demo"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- [**Live Demo**](https://dev--superb-taiyaki-212be8.netlify.app/)
+- [**Video Presentation**](https://www.loom.com/share/5576a817a77745728d3db2d78484d70a)
```

Comment:
- [ ] Your loom video currently is not opening up. Kindly recheck the link and ensure that it is working

Reply:
ok solved @frank1738